hivemq-edge/src/frontend/src/components/rjsf/__internals/TopicInputTemplate.tsx

```diff
@@ -23,7 +24,25 @@ export const TopicInputTemplate: FC<BaseInputTemplateProps> = (props) => {
       isInvalid={rawErrors && rawErrors.length > 0}
     >
       <FormLabel htmlFor={id}>{label}</FormLabel>
-      <SingleTopicCreatableSelect isLoading={isLoading} options={data} id={id} value={value} onChange={onChange} />
+      {!multiple && (
+        <SingleTopicCreatableSelect
+          isLoading={isLoading}
+          options={data}
+          id={id}
+          value={value}
+          onChange={onChange}
+          isCreatable={Boolean(create !== false)}
```

Comment:
This would imply that `create` being `undefined`, `null` or `"whatever"` would make it creatable. Maybe there are already conditions preventing this from happening?
Wouldn't it be clearer to use `create === true`? Or is it done this way so that you can pass `1` or `true` or `'true'`?

Reply:
Good question. It's also to be true if undefined, so something like 
```
          isCreatable={create === true || create === undefined}

```
But indeed, other values will interfere.